apps/system/test/unit/task_manager_test.js

```diff
@@ -94,6 +94,16 @@ suite('system/TaskManager >', function() {
     // Then dispatch the cardviewshow event after a tick
     clock.tick();
   }
+  function hideTaskManager(clock) {
+    window.dispatchEvent(new CustomEvent('taskmanagerhide'));
+
+    // the cardviewbeforeclose event after a tick
+    // the cardviewclosed event after a tick
+    clock.tick(2);
+    // Clean up DOM in case a test didn't end up calling hide/_removeCards
+    cardsList.innerHTML = '';
+    cardsView.className = '';
+  }
```

Comment:
Can we just make this a global `teardown()`?
My reasoning is: I don't want these last 2 lines to have side effects during a test. Currently they won't since `hideTaskManager` is always called as part of a `teardown`, but I don't want somebody to mistakenly use this function as part of a test later just to test a real task manager hide action.